jipdate.py

```diff
@@ -337,6 +341,16 @@ def create_default_config(config_file):
 # Extra comments added to each Jira issue (multiline is OK)
 comments:
         - "No updates since last week."
+
+# Header of the file (multiline is OK). It will be followed by JIRA_USERNAME
+header:
```

Comment:
ok.
Do you want me to submit a new version or you will handle the changes will merging ?